scripts/install-kusion.sh

```diff
@@ -22,7 +22,7 @@ for arg in "$@"; do
     fi
```

Comment:
The new implementation using `cat` and `tr` is less efficient than the original `base64 -w 0`. The original command directly outputs base64 without newlines, while the new version uses two additional processes (`cat` and `tr`) and a pipe. Consider reverting to the original implementation for better performance.